events/2022.12.Online/TD/manual-csv-changelogs/node-wot.md

```diff
@@ -44,38 +43,78 @@
 
 ## LINE-CHANGE
 
-- `privacy-essential-metadata-only` was moved from Line52 to 51
-- `privacy-immutable-id-as-property` was moved from Line54 to 52
-- `privacy-mutable-identifiers` was moved from Line56 to 53
-- `privacy-temp-id-metadata` was moved from Line58 to 54
-- `sec-body-name-json-pointer-type` was moved from Line61 to 55
-- `sec-inj-no-intl-markup` was moved from Line62 to 56
-- `security-context-secure-fetch` was moved from Line66 to 57
-- `security-jsonld-expansion` was moved from Line67 to 58
-- `security-mutual-auth-td` was moved from Line68 to 59
-- `security-no-execution` was moved from Line69 to 60
-- `security-oauth-limits` was moved from Line70 to 61
-- `security-remote-context` was moved from Line71 to 62
-- `security-server-auth-td` was moved from Line72 to 63
-- `security-static-context` was moved from Line73 to 64
-- `td-context-ns-td10-namespacev10` was moved from Line75 to 65
-- `td-default-AdditionalResponseContentType` was moved from Line76 to 66
-- `td-default-observable` was moved from Line77 to 67
-- `td-security-extension` was moved from Line79 to 70
-- `td-text-direction-first-strong` was moved from Line83 to 71
-- `td-text-direction-language-tag` was moved from Line84 to 72
-- `thing-model-td-generation-processor-extends` was moved from Line85 to 73
-- `thing-model-td-generation-processor-forms` was moved from Line86 to 74
-- `thing-model-td-generation-processor-imports` was moved from Line87 to 75
-- `thing-model-td-generation-processor-placeholder` was moved from Line88 to 77
-- `thing-model-td-generation-processor-required` was moved from Line89 to 78
-- `thing-model-td-generation-processor-type` was moved from Line90 to 79
-- `tm-overwrite-interaction` was moved from Line95 to 81
-- `tm-overwrite-types` was moved from Line96 to 82
-- `tm-placeholder-replacement` was moved from Line97 to 83
-- `tm-ref-recursive-extensions` was moved from Line101 to 84
-- `tm-tmRef-overwrite-possibility` was moved from Line102 to 85
-- `tm-tmRef-overwrite-process` was moved from Line103 to 86
-- `tm-tmRef-overwrite-semantic-meaning` was moved from Line104 to 87
-- `tm-tmRef2` was moved from Line106 to 88
+- `td-default-alg` was moved from Line13 to 12
```

Comment:
Nit: I noticed the tool misses to add a space before the "from" line
Line13 to 12
could be
Line 13 to 12

Reply:
Fixing now, please see https://github.com/thingweb/thingweb-playground/pull/428